core/src/main/java/org/infinispan/configuration/serializing/Serializer.java

```diff
@@ -250,6 +248,7 @@ private void writeCacheContainer(XMLExtendedStreamWriter writer, ConfigurationHo
             default:
                break;
          }
+         writeExtraConfiguration(writer, config.modules());
```

Comment:
We may want to include this in a separate PR. I just worry if this is tested, since xml schema validation can be ordered.

Reply:
Curious that I did not change it. Intellij did this for some bizarre reason...

Reply:
It was probably a stale merge from https://github.com/infinispan/infinispan/commit/adff9c9508879e453b19af35aab4b637d05ddc0d